quadriga/pom.xml

```diff
@@ -44,7 +44,10 @@
         <citesphere.base.url></citesphere.base.url>
         <citesphere.client.id></citesphere.client.id>
         <citesphere.client.secret></citesphere.client.secret>
-
+        
+        <conceptpower.base.url></conceptpower.base.url>
+        <conceptpower.id.endpoint></conceptpower.id.endpoint>
```

Comment:
is that used anywhere?

Reply:
It's value is currently defined in config.properties. Shall I remove this entry from pom.xml or give the value here?

Reply:
remove it here